Health-and-wellness-mobile/src/views/login_view.scss

```diff
@@ -0,0 +1,21 @@
+.content {
+    align-content: center;
+}
+.image {
+    display: block;
+    margin-left: auto;
+    margin-right: auto;
+    margin-top: 10px;
```

Comment:
if you use the @import '../../theme/theme.scss' you can use the $pad-lg for margin top instead just so we can dynamically adjust padding if we need to;